app/controllers/participants_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ParticipantsController < ApplicationController
+  def show
+    @participant = current_site.participants.find(params[:id])
+    @posts = @participant.credits.map(&:post)
```

Comment:
This `@posts` is just an `Array`, not an A`R::Relation`.
This blocks using pagination, scope chaining or some AR specific usage.

Can you add `has_many :posts, through: :credits` relation to `Participant` to allow using AR methods?